compiler/rustc_target/src/spec/wasm64_unknown_unknown.rs

```diff
@@ -23,11 +23,15 @@ pub fn target() -> Target {
     // For now this target just never has an entry symbol no matter the output
     // type, so unconditionally pass this.
     clang_args.push("-Wl,--no-entry".to_string());
-    options
-        .pre_link_args
-        .get_mut(&LinkerFlavor::Lld(LldFlavor::Wasm))
-        .unwrap()
-        .push("--no-entry".to_string());
+
+    let lld_args = options.pre_link_args.get_mut(&LinkerFlavor::Lld(LldFlavor::Wasm)).unwrap();
+    lld_args.push("--no-entry".to_string());
+    lld_args.push("-mwasm64".to_string());
+
+    // Any engine that implements wasm64 will surely implement the rest of these
+    // features since they were all merged into the official spec by the time
+    // wasm64 was designed.
+    options.features = "+bulk-memory,+mutable-globals,+sign-ext,+nontrapping-fptoint".to_string();
```

Comment:
Is this actually guaranteed and documented in the spec? Is there ever a reason an implementation might choose (for instance) to build a very simple wasm64 implementation, such as an interpreter, but not support some of these features?

This may still be a reasonable default, but it seems worth documenting, perhaps in target documentation for `wasm64-unknown-unknown`.

Reply:
To be clear, I'm *not* suggesting that the features themselves need documentation here. I'm exclusively suggesting that the above string of features should be mentioned in the docs somewhere, and that's it. Just so that people know that's being passed to the backend without having to dig into the source code to find that out.